tests/examples/sql_inject_target/factureconf.py

```diff
@@ -92,6 +92,7 @@ def conf_targets():
     return [
         {
             'name': 'actors',
+            'format': 'select_unionall',
```

Comment:
Oh yeah, this definitely makes more sense in the targets than what I was thinking initially (the table definition).  Nice!